lib/spree_signifyd.rb

```diff
@@ -20,7 +20,7 @@ def set_score(order:, score:)
 
   def approve(order:)
     order.contents.approve(name: self.name)
-    order.shipments.each { |shipment| shipment.ready! unless shipment.ready? }
+    order.shipments.each(&:ready!)
     order.updater.update_shipment_state
     order.save!
   end
```

Comment:
+1 to this. How does Signifyd know that there isn't something else in the chain that needs to run prior to ready-ing the shipments?  It seems like somehow this ought to be code that lives in the app rather than in solidus_signifyd.